.github/workflows/config.yml

```diff
@@ -27,10 +27,10 @@ jobs:
           fetch-depth: 0
       - name: Write variables to .env file
         run: |
-          if [ -f ".env" ]
+          if [ ! -f ".env" ]
```

Comment:
is this how you do negation?

Reply:
yeah but i mean is this valid bash?

Reply:
i believe so, i've tested the if statement locally